application-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -226,6 +226,8 @@
   float: right;
   position: relative;
   z-index: 2;
+  margin-left: 1.5em;
+  margin-bottom: 0.5em;
```

Comment:
I used a smaller margin because it only shifts the second line of content. The least possible margin which could have done it was 0.4em but I chose 0.5em(not intentional). Margin as big as 1.1em was also doing the same thing(shifting only second line of blog). I think that if I choose a bigger value then it will shift third line of content also. I wil certainly look bad.